programs/ziti-edge-tunnel/instance-config.c

```diff
@@ -176,4 +179,15 @@ void cleanup_instance_config() {
     } else {
         ZITI_LOG(ZITI_WTF, "Could not clean instance config. The semaphore is not initialized.");
     }
+}
+
+char* get_config_file_name() {
+    if (config_dir != NULL) {
+        char* config_file_name = calloc(FILENAME_MAX, sizeof(char));
```

Comment:
leak

Reply:
this is a bad practice -- since it's sometimes allocated that caller has no idea if it can be freed